frontend/components/Documents.js

```diff
@@ -17,6 +23,67 @@ const Documents = () => {
             });
     }, []);
 
+    const handleTitleInputChange = (event) => {
+	    event.persist();
```

Comment:
Nice job accommodating the idiosyncrasies of React's `SyntheticEvent` and the async state setters with this `event.persist()`, but we actually don't need `event.persist()` any longer. With the new project, we've upgraded to React v17, where the `SyntheticEvent` is no longer pooled (per the **note** on [this page](https://reactjs.org/docs/events.html)). You can remove it here and elsewhere in the component.

Reply:
It's been removed!